### PR TITLE
[WIP] Scikit Opt Wrapper

### PR DESCRIPTION
**Pull request by dendisuhubdy**

### CODE REVIEW

**Comment by dendisuhubdy** [comment]
@bouthilx oh God, really?

**Comment by dendisuhubdy** [comment]
@bouthilx it shouldn't be, I'll figure it out

File: setup.py [hunk 1]
```diff
@@ -1,25 +1,25 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-"""Installation script for `metaopt.algo.skopt.bayes`."""
+"""Installation script for `orion.algo.skopt.bayes`."""
 from setuptools import setup
 
 setup_args = dict(
-    name='metaopt.algo.skopt.bayes',
+    name='orion.algo.skopt.bayes',
```
Comment (tsirif) [inline setup.py]: Shall we keep the convention that the PyPI package name is going to be the same as the Python package itself? So `orion.algo.skopt`?

File: setup.py [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
     version=0.1,
     description="Implement a wrapper for skopt bayesian optimizer, for demo and testing.",
     license='BSD-3-Clause',
     author='Xavier Bouthillier',
     author_email='xavier.bouthillier@umontreal.ca',
-    url='https://github.com/bouthilx/metaopt-skopt-bayes',
-    packages=['metaopt.algo.skopt'],
+    url='https://github.com/bouthilx/orion-skopt-bayes',
```
Comment (tsirif) [inline setup.py]: url here is wrong

Comment (tsirif) [reply setup.py]: Well it was on me, I would have Python package name, to be the same as PyPI package name, same as repo name.

Comment (bouthilx) [reply setup.py]: You mean orion.algo.skopt and orion-algo-skopt?

Comment (bouthilx) [reply setup.py]: Let's favor coherence, I agree

**Pull request merged**